cmd/tracker/doctor.go

```
@@ -137,588 +115,136 @@ func runDoctorWithConfig(workdir string, cfg DoctorConfig) error {
-			printCheck(false, ".ai is not a directory")
-			allOk = false
-		} else if !isDirWritable(aiDir) {
-			printCheck(false, fmt.Sprintf("%s exists but is not writable", aiDir))
-			printHint(fmt.Sprintf("check permissions: chmod u+w %s", aiDir))
-			allOk = false
-		} else {
-			printCheck(true, fmt.Sprintf("%s exists and is writable", aiDir))
-		}
+	// Append missing entries (best-effort; silent on error — the user
+	// already saw the warning).
+	suffix := "\n"
```

golangci-lint (2.11.4)

[error] ineffectual assignment to suffix

(ineffassign)